README.md

```diff
@@ -49,9 +49,12 @@ make osm ARGS='-p http://download.geofabrik.de/europe/monaco-latest.osm.pbf -H p
 Now you're all set! You should be able to execute tegola and generate some tiles or kartotherian to see the tiles you have generated.
 
 ```
+make up
 make run_kartotherian
 ```
 
+In your MediaWiki installation you can now set `$wgKartographerMapServer` to `http://localhost:6533`. Make sure your wiki's host name and `$wgServer` isn't "localhost", otherwise Kartotherian won't be able to call back to your wiki.
```

Comment:
I didn't notice earlier, but these instructions only work for bare metal.  Much better to suggest one of the FQDNs such as `http://maps.local.wmftest.net:6533`, which will resolve to localhost on bare metal, but can be overridden to point at the docker host when run inside of a container.

Reply:
Please feel free to expand it to cover both possibilities. I just wanted to document the pitfalls I found.

And can we please stop using the term "bare metal"? It starts to sound like an insult to me. A setup that uses Docker is not "bare metal".